.github/workflows/upgrade-testing.yaml

```diff
@@ -0,0 +1,50 @@
+name: Upgrade Testing
+
+on:
+  push:
```

Comment:
Isn't push too late?

Reply:
@k0da , when it would start? I suppose push is just right time.

Reply:
I thought about PR, but it would start once pushed to testing branch? Would result be vissible under PR checks?

Reply:
@k0da , yes the same as terratest

Reply:
ic